src/simple.jl

```diff
@@ -121,12 +122,16 @@ function cvode(f::Function, y0::Vector{Float64}, t::Vector{Float64}, userdata::A
         tout = [0.0]
         for k in 2:length(t)
             flag = @checkflag CVode(mem, t[k], ynv, tout, CV_NORMAL)
+            if !callback(ynv)
```

Comment:
Maybe give it the current time `t[k]` and the `mem` to allow interpolations (that's all that should be needed for `CVodeGetDky`, right?)? And add it to the docstring.